pkg/volume/volume.go

```diff
@@ -28,6 +28,22 @@ import (
 type Volume interface {
 	// GetPath returns the directory path the volume is mounted to.
 	GetPath() string
+
+	Accountable
+}
+
+type VolumeDefaults struct {
+	AccountingNil
```

Comment:
Instead of returning `-1`, should we instead return an established error message?